src/command/render/render.ts

```diff
@@ -918,13 +919,26 @@ async function resolveFormats(
   project?: ProjectContext,
 ): Promise<Record<string, Format>> {
   // merge input metadata into project metadata
-  const projMetadata = await projectMetadataForInputFile(
+  const projMetadataForInput = await projectMetadataForInputFile(
     target.input,
     options.flags,
     project,
   );
   const inputMetadata = target.metadata;
 
+  // look for directory level metadata and merge that into the project
+  // metadata (this will still allow document level metadata to overwrite it)
+  const directoryMetadataForInput = project?.dir
+    ? directoryMetadataForInputFile(
+      project.dir,
+      dirname(target.input),
+    )
+    : {};
+  const projMetadata = mergeConfigs(
+    projMetadataForInput,
+    directoryMetadataForInput,
+  );
+
```

Comment:
There is another consideration here concerning the logic for ordering the formats. Let's discuss realtime on Monday.